services/src/test/scala/cromwell/services/keyvalue/impl/setup_test_dbs.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# setup_test_dbs.sh:
+# The tests in KeyValueDatabaseSpec assume a number of test databases will be running locally. The
+# tests print out a message on how to set these up: This script just runs all of them in sequence
+# with some retries in case the docker images aren't available when the DB reset is run.
```

Comment:
ToL: This bash is ok, but the tests should perhaps consider using [TestContainers](https://github.com/testcontainers/testcontainers-scala#readme) (java or scala versions) at some point.

Future PR.

Reply:
Oh, this wasn't actually intended to get checked in, since it's yet another thing to keep sync'd... hmm, I guess it's not actively evil, but it's probably less useful than it could/should be

Reply:
I'll add a big scary disclaimer

Reply:
Might this be implemented more parsimoniously with a `crmdbms` flag akin to `crmdbg` and `crmit` that would leverage the existing CI code to bring up the databases for us?

Reply:
I'll just remove this file. It's not really important to this PR and is rightly controversial.